### ISSUE TITLE
Create request overview

### ISSUE THREAD

**Issue by mdp97** [amosproj/amos2021ws02-agile-trans-jira#36]
Create a visual representation of the request overview. It should be implementable on the main page. No functionality necessary yet. 

## User story
1. As a product owner/scrum master
2. I need an overview on all the requests ever created
3. So that I can evaluate the current project status

## Acceptance criteria
* All requests are seeable
* Filtering on specific criteria possible

## Definition of done
* Added only after week 5
* The same for all features

**Comment by emiliakaf** [comment]
The visual representation of the Request Overview has been implemented in the Request Overview Tab (for code see main_page.vm).
![image](https://user-images.githubusercontent.com/92568413/142393112-e6de6a40-fc52-4972-aab9-ee98c4fa9d5a.png)

For the access through the Request Overview I have an idea which is not part of the mockup, but would contribute to a better usability and will discuss with the team in the meeting.
